exercises/practice/acronym/src/acronym.lfe

```diff
@@ -0,0 +1,5 @@
+(defmodule acronym
+    (export (abbreviate 1)))
+
+(defun abbreviate (str)
+  (throw "You need to implement abbreviate"))
```

Comment:
Suggest rewording to something friendlier like "Please implement the abbreviate function".